src/pages/categories/index.js

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
+import { useSelector } from 'react-redux';
+import './styles/categoriesPage.css';
 
-const Categories = () => (
-  <div>
-    <h1>Categories</h1>
-  </div>
-);
+const Categories = () => {
+  const title = useSelector((state) => state.categoriesReducer.status);
+
+  return (
+    <section className="categories-section">
+      <h1>{title}</h1>
+    </section>
+  );
```

Comment:
- [ ] Kindly add a button with the name `Check status` and on the click event of that button, dispatch the corresponding action as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#:~:text=In%20your%20React%20component%20responsible%20for%20categories,you%20created%20before%20and%20use%20it%20here). Instead of dispatching it inside the `navbar` component.